DRAFT.md

```diff
@@ -1589,6 +1583,28 @@ such types are copy constructible. We make `polymorphic` unconditionally copy
 and move constructible. The authors do not envisage that this could be relaxed
 in a future version of the C++ standard.
 
+While a copy constructor cannot be a template, we can conditionally constrain
+copy construction of `indirect`, in C++20 and later, by defining:
+
+```c++
+indirect(const indirect& other) requires false = delete;
+
+template <typename TT=T>
```

Comment:
template <typename TT = T>

Reply:
I'm not sure what you mean? I've possibly missed the same typo twice in my head.

`indirect(const indirect& other) requires false = delete;` is not a function template it's a non-template function from a class template (which means we can constrain it).

Reply:
Missing spacing discussed locally and fixed.